auv_setup/config/robots/gladlaks.yaml

```diff
@@ -46,8 +46,8 @@ propulsion:
       thrust:      [-34.5, -24.0, -12.9, -3.92, 0.00, 4.91, 12.6, 26.1, 35.5, 44.3] 
       pulse_width: [ 1100,  1192,  1292,  1400, 1500, 1600, 1676, 1780, 1840, 1900]
     map: [0, 1, 2, 3, 4, 5, 6, 7]  
-    direction: [1, 1, 1, 1, 1, 1, 1, 1]
-    offset: [0.00,  0.00,  0.00, 0.00, 0.00,  0.00,  0.00, 0.00]
+    direction: [1, -1, 1, 1, -1, -1, 1, -1]
+    offset: [80, 80, 80, 80, 80, 80, 80, 80] # Offset IN PWM!
```

Comment:
Last time I worked on this, the offset was in Newton. Where in the code is the change from Newton to PWM reflected?

Reply:
We fixed a bug in the thruster interface and changed to offset the PWM on the last Saturday of the pool test. The change should only affect the thruster interface, although manta.yaml definitely is outdated now

Reply:
Alright, cool.